docs/actions.rst

```diff
@@ -55,8 +55,8 @@ Run an Ansible playbook.
    * - var_root
      - If the event is a deeply nested dictionary, the var_root can specify the key name whose value should replace the matching event value. The var_root can take a dictionary to account for data when we have multiple matching events.
      - No
-   * - rulebook_extra_vars
-     - In addition to the passed in vars and matching events the rulebook_extra_vars would be passed into the playbook as extra vars.
+   * - extra_vars
+     - Additional vars to be passed into the playbook as extra vars.
```

Comment:
@bzwei Should we document the precedence here, also QE would have to validate that. If we pass in --variables here they would override the variables defined in the rulebook in the extra_vars section. Maybe we can add a test for this and use the validate_args_playbook

Reply:
@bzwei vars can be passed in a file as well as from the environment vars.
https://github.com/ansible/ansible-rulebook/blob/a6c031f15b12c1b75a2574a19f03da6fde3bf783/ansible_rulebook/app.py#L126

Reply:
@bzwei ansible uses extra-vars with a dash instead of an underscore. https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#defining-variables-at-runtime Shold we align the rulebook to follow the same. Also I am thinking we should changes the ansible-rulebook to use -e or --extra-vars instead of  --variables

Reply:
totally agree with renaming extra-vars flag.